internal/lib/stop.go

```diff
@@ -11,6 +10,15 @@ import (
 
 // ContainerStop stops a running container with a grace period (i.e., timeout).
 func (c *ContainerServer) StopContainer(ctx context.Context, ctr *oci.Container, timeout int64) error {
+	cStatus := ctr.StateNoLock()
```

Comment:
if I use` ctr.State()`, the it case will failed, I think there is bug in `c.runtime.UpdateContainerStatus` which take the lock for long time. I will find it out latter